site/src/api/typesGenerated.ts

```diff
@@ -1908,3 +1908,92 @@ export const WorkspaceTransitions: WorkspaceTransition[] = [
 
 // From codersdk/workspaceproxy.go
 export type RegionTypes = Region | WorkspaceProxy
+
+// Code generated by 'make site/src/api/typesGenerated.ts'. DO NOT EDIT.
+
+// From healthcheck/accessurl.go
+export interface HealthcheckAccessURLReport {
+  readonly access_url: string
+  readonly healthy: boolean
+  readonly reachable: boolean
+  readonly status_code: number
+  readonly healthz_response: string
+  readonly error?: string
+}
+
+// From healthcheck/derp.go
+export interface HealthcheckDERPNodeReport {
+  readonly healthy: boolean
+  // Named type "tailscale.com/tailcfg.DERPNode" unknown, using "any"
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any -- External type
+  readonly node?: any
+  // Named type "tailscale.com/derp.ServerInfoMessage" unknown, using "any"
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any -- External type
+  readonly node_info: any
+  readonly can_exchange_messages: boolean
+  // This is likely an enum in an external package ("time.Duration")
+  readonly round_trip_ping: number
+  readonly uses_websocket: boolean
+  readonly client_logs: string[][]
+  readonly client_errs: string[][]
+  readonly error?: string
+  readonly stun: HealthcheckDERPStunReport
+}
+
+// From healthcheck/derp.go
+export interface HealthcheckDERPRegionReport {
+  readonly healthy: boolean
+  // Named type "tailscale.com/tailcfg.DERPRegion" unknown, using "any"
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any -- External type
+  readonly region?: any
```

Comment:
Are these json fields uppercase? I don't see json tags on the tailcfg struct.

So should these be in our api? I assume these apis are user faceing?

Reply:
Yeah they are, but they're mostly just for human consumption in the debug menu.

Reply:
Do we need these at all in the typesgenerated then? Is the debug menu in the FE? And does it access any of these fields?

Reply:
Bruno is writing a page in the FE for the healthcheck, and would like to be able to parse some of the output. The Tailscale internal stuff isn't too important currently.